web/src/games/barnbarter/index.ts

```diff
@@ -0,0 +1,24 @@
+const Thumbnail = require('./media/thumbnail.jpg');
+import { GameMode } from 'gamesShared/definitions/mode';
+import { IGameDef, IGameStatus } from 'gamesShared/definitions/game';
+import instructions from './instructions.md';
+
+export const BarnBarterDef: IGameDef = {
+  code: 'barnbarter',
+  name: 'Barn Barter',
+  contributors: ['Spooky-O'],
+  minPlayers: 1,
```

Comment:
Hummm 1 player would be local mode, right? You wouldn't need to go over the network to play. So this should be 2 and add the GameMode.Local to the modes list below